shared/signup/phone-number/phone-input.tsx

```diff
@@ -220,6 +220,7 @@ type Props = {
   onChangeNumber: (phoneNumber: string) => void
   onChangeValidity: (valid: boolean) => void
   onEnterKeyDown?: () => void
+  result?: React.ReactNode
```

Comment:
The design calls for having the user appear aligned with the phone number input box. Rather than using a ref, this was suggested in #frontend.